### DISCUSSION TITLE
Add Fanboy Enhanced tracking list? [CLOSED]

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#1866] [Ideas]
Hello uBlock Origin developers and contributors,

Today I have came across a new enhanced tracker list from Fanboy, which is called "Fanboy Enhanced Tracking List" which is the supplemental list of EasyPrivacy and Easylist.

The list URL is at `https://secure.fanboy.co.nz/enhancedstats.txt&title=Enhanced%20Tracking`
Maybe we can make it as an optional list for users to choose.

**Comment by u-RraaLL** [comment]
> Updated: 18 Feb 2021

**Comment by ghost** [comment]
https://github.com/gorhill/uBlock/commit/4320e9c28b708f179787b1fb93588b65ed6a5b52

>Too likely to cause breakage. It was originally added without
having been really evaluated. As per feedback, it's too likely
to cause breakage.

>The list is seemingly used to evaluate filters, which are moved
to EasyPrivacy once they are deemed valid and useful.

**Comment by ghost** [comment]
OK case closed then
Sorry!
I will remove that custom filter and stick to the ones uBlock origin originally provides.
Thank you for the devlopment
